crates/sui-core/src/authority.rs

```diff
@@ -1045,18 +1048,11 @@ impl AuthorityState {
                 .expect("Cannot bulk insert genesis objects");
         }
 
-        let committee = if epoch_store.database_is_empty() {
-            epoch_store
-                .init_genesis_epoch(genesis_committee.clone())
-                .expect("Init genesis epoch data must not fail");
-            genesis_committee
-        } else {
-            epoch_store
-                .get_latest_authenticated_epoch()
-                .epoch_info()
-                .committee()
-                .clone()
-        };
+        let committee = epoch_store
```

Comment:
Here we get the very last committee, which I think is the one compatible with the latest written stuff in the DB when we start. However, this may not be the actual latest committee of the node has been away for a while. Do we pass that in a different place, or do we only need this when we sync?

Reply:
We don't have that support yet. We will need to pass some seed node addresses at start for the node to sync.